src/Common/src/TypeSystem/Ecma/EcmaMethod.cs

```diff
@@ -415,5 +416,33 @@ public override PInvokeMetadata GetPInvokeMethodMetadata()
 
             return new PInvokeMetadata(moduleName, name, (PInvokeAttributes)import.Attributes);
         }
+
+        public override PInvokeParameterMetadata[] GetPInvokeParameterMetadata()
+        {
+            if (!IsPInvoke)
```

Comment:
Okay, I see. In that case should I rename PInvokeParameterMetadata -> ParameterMetadata?